packages/web-components/src/components/content-item-horizontal/content-item-horizontal-thumbnail.scss

```diff
@@ -0,0 +1,148 @@
+  :host(#{$dds-prefix}-content-item-horizontal-thumbnail) {
+    @include carbon--make-row;
+
+    border-top: 1px solid $ui-03;
+    margin: 0;
+    padding-left: $carbon--spacing-05;
+    padding-right: $carbon--spacing-05;
+
+    @include carbon--breakpoint(sm) {
+      padding-top: $carbon--spacing-07;
+      padding-bottom: $carbon--spacing-07;
+    }
+
+    @include carbon--breakpoint(md) {
+      min-height: rem(306px);
+      padding-top: $carbon--spacing-07;
+      padding-bottom: $carbon--spacing-10;
+    }
+
+    @include carbon--breakpoint(lg) {
+      min-height: rem(272px);
+    }
+
+    @include carbon--breakpoint(xlg) {
+      min-height: rem(252px);
+    }
+  }
+
+  // Layout
+  .#{$prefix}--content-item-horizontal-thumbnail__row {
+    @include carbon--make-row;
+  }
+
+  .#{$prefix}--content-item-horizontal-thumbnail__col {
+    @include carbon--make-col-ready;
+
+    &--1 {
+      @include carbon--breakpoint(sm) {
+        @include carbon--make-col(4, 4);
+      }
+
+      @include carbon--breakpoint(md) {
+        @include carbon--make-col(4, 8);
+      }
+
+      @include carbon--breakpoint(lg) {
+        @include carbon--make-col(8, 12);
+        @include carbon--make-row;
+
+        margin: 0;
+      }
+    }
+
+    &--2 {
+      @include carbon--breakpoint(sm) {
+        @include carbon--make-col(4, 4);
+      }
+
+      @include carbon--breakpoint(md) {
+        @include carbon--make-col(4, 8);
+
+        display: flex;
+        justify-content: flex-end;
+      }
+
+      @include carbon--breakpoint(lg) {
+        @include carbon--make-col(4, 12);
+
+        display: flex;
+      }
+    }
+  }
+
+  .#{$prefix}--content-item-horizontal-thumbnail__heading-wrapper,
+  .#{$prefix}--content-item-horizontal-thumbnail__content-wrapper {
+    @include carbon--breakpoint(md) {
+      padding-right: $carbon--spacing-07;
+    }
+    @include carbon--breakpoint(lg) {
+      @include carbon--make-col(4, 8);
+    }
+  }
+
+  .#{$prefix}--content-item-horizontal-thumbnail__heading-wrapper {
+    @include carbon--breakpoint(lg) {
+      padding-right: $carbon--spacing-05;
+    }
+  }
+
+  .#{$prefix}--content-item-horizontal-thumbnail__content-wrapper {
+    @include carbon--breakpoint(lg) {
+      padding-left: $carbon--spacing-05;
+      padding-right: $carbon--spacing-07;
+    }
+  }
+
+  .#{$prefix}--content-item-horizontal-thumbnail__thumbnail-wrapper {
+    @include carbon--make-col(3, 4);
+  }
+
+  // Content / typography
+  :host(#{$dds-prefix}-content-item-horizontal-thumbnail) ::slotted([slot='heading']) {
+    @include carbon--type-style('expressive-heading-03', true);
+
+    color: $text-01;
+    display: block;
+    margin-bottom: $carbon--spacing-07;
+    width: 100%;
+  }
+
+  :host(#{$dds-prefix}-content-item-horizontal-thumbnail-copy) ::slotted(:not([slot])) {
+    margin-bottom: $carbon--spacing-07;
+
+    p {
+      color: $text-01;
+    }
+  }
+
+  :host(#{$dds-prefix}-link-list) ::slotted([slot='footer']) {
```

Comment:
I think this may be changing the layout of components using this element, can you contain it to just the `content-item-horizontal-thumbnail`?